src/Compilers/CSharp/Portable/Compilation/CSharpCompilation.cs

```diff
@@ -210,7 +210,7 @@ public override INamedTypeSymbol CreateErrorTypeSymbol(INamespaceOrTypeSymbol co
         /// </summary>
         public static CSharpCompilation CreateSubmission(
             string assemblyName,
-            SyntaxTree syntaxTree = null,
+            IEnumerable<SyntaxTree> syntaxTrees = null,
```

Comment:
I'm actually going to revert this particular change (@tmat had a better idea)...update coming later today...